posts/models.py

```diff
@@ -0,0 +1,68 @@
+import uuid
+from django.db import models
+from authors.models import Author
+from django.utils.translation import gettext_lazy as _
+
+
+class Comment(models.Model):
+    id = models.UUIDField(primary_key=True, editable=False, default=uuid.uuid4)
+    author = models.ForeignKey(Author, related_name="commenter", on_delete=models.CASCADE)
+    comment = models.TextField()
+    published = models.DateTimeField()
+
+    # TODO: question: should comments supports the same content-types as posts?
+    # content_type = ?
+
+class Post(models.Model):
+    # https://docs.djangoproject.com/en/3.2/ref/models/fields/#enumeration-types
+    class ContentType(models.TextChoices):
+        MARKDOWN = 'MDN', _('text/markdown')
+        PLAIN = 'PLN', _('text/plain')
+        APPLICATION = 'APP', _('application/base64')
+        IMAGE_PNG = 'PNG', _('image/png;base64')
+        IMAGE_JPEG = 'JPG', _('image/jpeg;base64')
+
+    class Visibility(models.TextChoices):
+        PUBLIC = 'PUB', _('PUBLIC')
+        FRIENDS = 'FRI', _('FRIENDS')
+        PRIVATE = 'PRI', _('PRIVATE')
+
+    id = models.UUIDField(primary_key=True, editable=False, default=uuid.uuid4)
+    url = models.URLField(editable=False)
+    author = models.ForeignKey(Author, related_name="post", on_delete=models.CASCADE)
+    
+    title = models.CharField(max_length=30) # title should not be super long
+    source = models.URLField(editable=False)
+    origin = models.URLField(editable=False)
+    description = models.CharField(max_length = 50)
+    content_type = models.CharField(max_length=3, choices=ContentType.choices, default=ContentType.PLAIN)
+    content = models.TextField()
+    published = models.DateTimeField()
+    comments = models.ManyToManyField(Comment, related_name="post_comments", blank=True)
```

Comment:
We need to link up Comment many---->one Post, many to many probably doesn't work. 

How about let's not have comments in Post, and instead, have Post as a foreign key in Comment, with this, posts can also have a related name, and that follows the same format as you have with Comment-Author relationship.

So basically, Comment is just a Author-Post relation table (two foreign keys), with extra comment body attached.

Reply:
done, Comment now have the Post foreign key